invenio_rdm_records/fixtures/data/vocabularies/roles.yaml

```diff
@@ -3,103 +3,124 @@
     datacite: ContactPerson
   title:
     en: Contact person
+    de: Kontaktperson
 - id: datacollector
   props:
     datacite: DataCollector
   title:
     en: Data collector
+    de: DatensammlerIn
 - id: datacurator
   props:
     datacite: DataCurator
   title:
     en: Data curator
+    de: DatenkuratorIn
 - id: datamanager
   props:
     datacite: DataManager
   title:
     en: Data manager
+    de: DatenmanagerIn
 - id: distributor
   props:
     datacite: Distributor
   title:
     en: Distributor
+    de: VerteilerIn
 - id: editor
   props:
     datacite: Editor
   title:
     en: Editor
+    de: EditorIn
 - id: hostinginstitution
   props:
     datacite: HostingInstitution
   title:
     en: Hosting institution
+    de: Bereitstellende Institution
 - id: producer
   props:
     datacite: Producer
   title:
     en: Producer
+    de: ProduzentIn
 - id: projectleader
   props:
     datacite: ProjectLeader
   title:
     en: Project leader
+    de: ProjektleiterIn
 - id: projectmanager
   props:
     datacite: ProjectManager
   title:
     en: Project manager
+    de: ProjektmanagerIn
 - id: projectmember
   props:
     datacite: ProjectMember
   title:
     en: Project member
+    de: Projektmitglied
 - id: registrationagency
   props:
     datacite: RegistrationAgency
   title:
     en: Registration agency
+    de: Registrierungsstelle
 - id: registrationauthority
   props:
     datacite: RegistrationAuthority
   title:
     en: Registration authority
+    de: Registrierungsbehörde
 - id: relatedperson
   props:
     datacite: RelatedPerson
   title:
     en: Related person
+    de: Verwandte Personen
 - id: researcher
   props:
     datacite: Researcher
   title:
     en: Researcher
+    de: WissenschafterIn
```

Comment:
"WissenschaftlerIn" (missing an l)

Reply:
as @utnapischtim mentioned elsewhere: https://www.onlinelektorat.at/1-000-und-1-fakt-aus-der-welt-des-lektorats-wissenschafter/
but as far as I know we are translating de_de, right? So WissenschaftlerIn with the l would be the way to go --- sorry for the Austrian and Swiss community...